src/test/scala/com/gd/twitteranalytics/TweetsDataLoaderTest.scala

```diff
@@ -0,0 +1,26 @@
+package com.gd.twitteranalytics
+
+import java.io.File
+import com.gd.twitteranalytics.util.AppConfigReader
+import com.holdenkarau.spark.testing.DataFrameSuiteBase
+import org.scalatest.FreeSpec
+
+class TweetsDataLoaderTest extends FreeSpec with DataFrameSuiteBase {
+
+  "Methods tests" - {
```

Comment:
should it be ` TweetsDataLoader` instead?

Reply:
Updated with class name instead of Methods.